save_to_db.py

```diff
@@ -0,0 +1,67 @@
+import auth
+import json
+import database_handling as udb
+from datetime import datetime
+import db_secrets
+
+def save_to_db(category_number, identifier):
+    db_secrets.get_db_secrets()
+    if identifier == 'minifigure_parts' or identifier == 'minifigure_prices':   
```

Comment:
you can also use an early return here

Reply:
I will have to split this to separate files due to integration into lambda functions, i guess